mlbench/master/api/utils/run_utils.py

```diff
@@ -54,7 +54,9 @@ def run_model_job(model_run, experiment="test_MPI"):
             '/conda/bin/python', "/codes/main.py",
             '--experiment', experiment,
             '--run_id',
-            model_run.id]
+            model_run.id,
+            '--config-file', '/codes/configs/debug_cifar10.json'
```

Comment:
You are right. I use the config files because applying changes to `run_model_job` seems not the same as others. Otherwise adding config files from dashboard/command line would be better.
The function body of `run_model_job` remain the same even after `helm upgrade`. Do you know if there are any ways to apply changes to `run_utils.py`-`run_model_job` easily?